strapi/strapi
# enhancement: display relations on history page

### What does it do?

- Displays the relations in the frontend using a custom input component, as the normal one does not work for our needs and the data we have
- Displays a warning alert when a relation or a media input is missing
- Fixes some UI issues:
  - apply bg color on the entire page
  - removes the sticky header we didn't want
  - adds some padding at the bottom of the page

### How to test it?

- Open the history for an entry with relations. You should see the relation's main field, or the document id as a fallback. Clicking on the name should take you to the edit view for that entry.
- Add relations to an entry, then delete the entries you just linked (don't just unlink them, go to their own page in the CM and delete them). You should see the "missing content" banner.
- Do the same but for media assets.

### Related issue(s)/PR(s)

This follows #20035 which was about the backend part.
